_ont/ont-zyxel-pmg3000-d20b.md

```diff
@@ -159,6 +172,7 @@ To restore the default combination you can try following [this method](https://g
 - [Not working with Broadcom BCM57810S](https://github.com/xvzf/zyxel-gpon-sfp/issues/10)
 - Issue on IPv6 discovery. But we are not sure if is a edge case of a particular ISP or not
 - [Some stick have a custom password](https://github.com/xvzf/zyxel-gpon-sfp/issues/6)
+- fw_setenv command doesn't work
```

Comment:
does not work or is not present, it seemed to me the second one

Reply:
admin@SFP:~# fw_setenv sgmii_mode 5
Cannot parse config file: No such file or directory
Error: environment not initialized

Reply:
OK is not present

Reply:
Like the new firmware of fs.com stick